.circleci/config.yml

```diff
@@ -370,6 +370,12 @@ jobs:
     steps:
       - test-linux:
           llvm: "11"
+  test-llvm11-go116:
+    docker:
+      - image: circleci/golang:1.16-rc-buster
```

Comment:
Can you update this to the final version? (Probably circleci/golang:1.16-buster)

Reply:
It wasn't available earlier... Maybe now.